runtime/common/src/paras_registrar.rs

```diff
@@ -609,7 +616,7 @@ mod tests {
 			assert_ok!(Registrar::register_parachain(
 				2u32.into(),
 				vec![3; 3].into(),
-				vec![3; 3].into(),
+				[0x00, 0x61, 0x73, 0x6d, 0x01, 0x00, 0x00, 0x00].to_vec().into(),
```

Comment:
why not use the const?

Reply:
Slight preference, although I don't mind using the const here.

Reply:
Changed!